src/flows/context/flow.list.tsx

```diff
@@ -222,7 +228,14 @@ export const FlowListProvider: FC = ({children}) => {
     }
     let id: string = `local_${UUID()}`
     try {
-      id = await createAPI(apiFlow)
+      const flow = await createAPI(apiFlow)
```

Comment:
yeah.. this is all hamajang. i think you need to have the flow run through the hydrate function before adding it places... and delete all of the modifying of _flow in here. `_flow = hydrate(flow)` should centralize the hydration logic and remove some bug around resource hooks not updating

Reply:
I totally hear what you're saying but I think this kind of goes along with the previous sentiment of trying to update hydrate to do all the things. It adds unnecessary complexity to the hydrate function that is entirely dependent on how a flow is structured. Updating this to be something like:

```
const notebook = await createAPI(apiFlow)
_flow = hydrate(notebook)
```

drops the local context that the DB has no concept about (like range, readOnly and refresh) which are properties of the spec (in this case not returned from the API). Let's sync up later